tools/robotick-launcher/tests/launcher/test_launcher_generate.py

```
@@ -38,12 +38,34 @@ def normalize_lines(lines):
+        text (str): Input text potentially containing absolute workload paths.
+    
+    Returns:
+        str: A copy of `text` where occurrences of paths ending with
+        `tests/test_data/robotick/robotick-core-workloads` are replaced with
+        `__WORKLOADS_ROOT__/tests/test_data/robotick/robotick-core-workloads`.
+    """
+    pattern = re.compile(r"[^ \n\r\t]*tests/test_data/robotick/robotick-core-workloads")
+    return pattern.sub("__WORKLOADS_ROOT__/tests/test_data/robotick/robotick-core-workloads", text)
 
 
 def _list_files(dir_path: Path):
```

Ruff (0.14.8)

Missing return type annotation for private function `_list_files`

(ANN202)